modules/rhcos-enabling-multipath.adoc

```diff
@@ -1,119 +1,50 @@
+. To enable multipath and start the `multipathd` daemon, run the following command:
 +
 [source,terminal]
 ----
-$ oc create -f ./99-master-kargs-mpath.yaml
+$ mpathconf --enable && systemctl start multipathd.service
 ----
+** Optional: If booting the PXE or ISO, you can instead enable multipath by adding `rd.multipath=default` from the kernel command line.
 
-. Check the machine configs to see that the new one was added:
-+
-[source,terminal]
-----
-$ oc get MachineConfig
-----
+. Edit the `/etc/multipath.conf` file to target the multipath device directly for installation, either on the kernel command line by using `coreos.inst.install_dev` or interactively by invoking the `coreos-installer` program.
 +
-.Example output
-[source,terminal]
-----
-NAME                                               GENERATEDBYCONTROLLER                      IGNITIONVERSION   AGE
-00-master                                          52dd3ba6a9a527fc3ab42afac8d12b693534c8c9   3.2.0             33m
-00-worker                                          52dd3ba6a9a527fc3ab42afac8d12b693534c8c9   3.2.0             33m
-01-master-container-runtime                        52dd3ba6a9a527fc3ab42afac8d12b693534c8c9   3.2.0             33m
-01-master-kubelet                                  52dd3ba6a9a527fc3ab42afac8d12b693534c8c9   3.2.0             33m
-01-worker-container-runtime                        52dd3ba6a9a527fc3ab42afac8d12b693534c8c9   3.2.0             33m
-01-worker-kubelet                                  52dd3ba6a9a527fc3ab42afac8d12b693534c8c9   3.2.0             33m
-99-master-kargs-mpath                              52dd3ba6a9a527fc3ab42afac8d12b693534c8c9   3.2.0             105s
-99-master-generated-registries                     52dd3ba6a9a527fc3ab42afac8d12b693534c8c9   3.2.0             33m
-99-master-ssh                                                                                 3.2.0             40m
-99-worker-generated-registries                     52dd3ba6a9a527fc3ab42afac8d12b693534c8c9   3.2.0             33m
-99-worker-ssh                                                                                 3.2.0             40m
-rendered-master-23e785de7587df95a4b517e0647e5ab7   52dd3ba6a9a527fc3ab42afac8d12b693534c8c9   3.2.0             33m
-rendered-worker-5d596d9293ca3ea80c896a1191735bb1   52dd3ba6a9a527fc3ab42afac8d12b693534c8c9   3.2.0             33m
-----
+You can configure multipath so that the names of the multipath devices are of the form `/dev/mapper/mpath` `_n_`. Without this setting, the names of the multipath devices are aliased to a World Wide Name (WWN) symlink, such as `/dev/disk/by-label`.
 
-. Check the nodes:
+. Append the kernel arguments by entering the following command:
 +
 [source,terminal]
 ----
-$ oc get nodes
+$ coreos-installer install --append-karg rd.multipath=default --append-karg root=/dev/disk/by-label/dm-mpath-root
```

Comment:
Is `by-label` sub dir okay here to stay in line with other examples in this section? Or should it really be `by-id`?

Reply:
This path is good. Once RHCOS is installed, the root partition and label exist so it's valid. Before installing RHCOS, it's just a random disk, so they have to use e.g. `/dev/mapper/` or `/dev/disk/by-id/wwn-...` to refer to the multipathed disk when installing. Which, speaking of, should this example include the target device to be clearer? E.g.

```
coreos-installer install /dev/disk/by-id/wwn-0xx194e957fcedb4841 --append-karg rd.multipath=default --append-karg root=/dev/disk/by-label/dm-mpath-root
```

Reply:
Sure, if that makes sense to you it's good with me. 😁  I changed the lead-in wording and the code block slightly to indicate that this is an example with a replaceable value. Do my edits make sense?